gradle.properties

```diff
@@ -9,6 +9,9 @@ asciidoctor_plugin_version=3.3.0
 axion_release_plugin_version=1.13.2
 buildtimetracker_version=0.11.0
 
+sb_webflux=2.7.4
+sb_security=2.7.4
```

Comment:
You can use the spring-boot-version below for these I believe. Not sure why you seperated these here instead of putting them in the bunch.